src/components/SvgWrapper/SvgWrapper.types.ts

```diff
@@ -1,6 +1,9 @@
-import * as React from 'react';
+export type SvgWrapperSize = 'small' | 'default' | 'big';
+export const svgWrapperSize = ['small', 'default', 'big'];
```

Comment:
Same question here, do we want to use the pluriel: `svgWrapperSizes`?

Reply:
Updated!